src/api/todos.ts

```diff
@@ -0,0 +1,14 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
+  return client.get<Todo[]>(`/todos?userId=${userId}`);
+};
+
+export const addTodo = (userId: number, todo: Todo) => {
+  return client.post<Todo>(`/todos?userId=${userId}`, todo);
+};
+
+export const deleteTodo = (userId: number, todoId: number) => {
+  return client.delete(`/todos/${todoId}?userId=${userId}`);
```

Comment:
here you dont need user Id 
<img width="1068" alt="Screenshot 2023-02-16 at 18 26 22" src="https://user-images.githubusercontent.com/34283676/219426981-bfb3bb77-c487-41f2-a5c8-68bc833745ff.png">